Snakefile

```diff
@@ -32,6 +32,8 @@ rule cluster_summary:
         "output/all_clusters.txt"
     params:
         levels=config['fastbaps']['levels']
+    log:
+        "logs/cluster_summary.log"
```

Comment:
Is this log file used/does it contain any output?

Reply:
oh yes there was a problem with the log file because for gubbins the directory is changes because gubbins creates file in `.` so then I would do:
`log:
        "../../../logs/gubbins_{strain}.log"`
but I am not sure if that is okay?

Reply:
I think that would be fine